components/responsive-img/responsive-img.css

```diff
@@ -0,0 +1,29 @@
+.images-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  width: 80%;
+  padding: 2rem;
+  margin: 1rem auto;
+  background-color: var(--comp-container-color);
+}
+
+.res-img {
+  width: 95%;
+  height: auto;
+}
+
+.res-round-img {
+  border-radius: 50%;
+  height: 35vh;
+  width: 35vh;
```

Comment:
Yeah, you're right. But again this will vary according to the img we are going to use on the screen. You might or might not need to set a height/ width. It also depends on the img dimensions.